src/adapters/IDB.js

```diff
@@ -81,6 +87,62 @@ export default class IDB extends BaseAdapter {
   }
 
   /**
+   * Batch operations.
+   *
+   * @param  {Function} fn The operation function, which must return a Promise.
+   * @return {Promise}
```

Comment:
Maybe explain that it returns a promise for `{operations, errors}`. What is the use of returning this data, is that for the unit tests?